browser/src/react-redux/App/LoginPage.js

```diff
@@ -0,0 +1,96 @@
+import React from 'react'
+import { connect } from 'react-redux'
+import { loginSuccess, loginFailure } from './User/updateUser.action'
+import axios from 'axios'
+
+const initialState = {
+  email: '',
+  password: '',
+  generalError: ''
+}
+
+class LoginPage extends React.Component {
+  state = initialState
+
+  handleChange = event => {
+    this.setState({
+      [event.target.name]: event.target.value
+    })
+  }
+
+  onSubmit = e => {
+    e.preventDefault()
+    this.setState(initialState)
+
+    const user = {
+      email: this.state.email,
+      password: this.state.password
+    }
+
+    axios
+      .post('/api/auth/login', user)
+      .then(res => {
+        this.props.loginSuccess(user)
```

Comment:
I don't think you're actually updating the user in redux. All you seemed to have been doing here was updating the success message and a property that tells us the user is logged in. We want to actually call the updateUser action here instead.